libraries/plugin.rb

```diff
@@ -14,8 +14,9 @@ def installed?(name, grafana_cli_bin)
       !cmd.stdout.split("\n").select { |item| item.include?('@') && item.match(name) }.empty?
     end
 
-    def build_cli_cmd(name, action, grafana_cli_bin)
-      "#{grafana_cli_bin} plugins #{action} #{name}"
+    def build_cli_cmd(name, action, grafana_cli_bin, plugin_url = nil)
+      extra_args = "--pluginUrl #{plugin_url} " if plugin_url
+      "#{grafana_cli_bin} #{extra_args}plugins #{action} #{name}"
```

Comment:
`extra_args` does not exist in this context?

Reply:
not sure i follow the question?

Reply:
I am seeing `"#{grafana_cli_bin} #{extra_args}plugins #{action} #{name}"` but in the inputs to the function I cannot see `extra_args` being passed in